packages/template-retail-react-app/app/ssr.js

```diff
@@ -59,6 +61,36 @@ const {handler} = runtime.createHandler(options, (app) => {
         })
     )
 
+    const createSlasHandler = () => {
+        const clientId = process?.env?.SLAS_PRIVATE_CLIENT_ID
+        const secret = process?.env?.SLAS_PRIVATE_CLIENT_SECRET
+        const shortCode = process?.env?.CC_SHORT_CODE
+        const target = `https://${shortCode}.api.commercecloud.salesforce.com`
```

Comment:
These environment variables are read either from your local terminal or from MRT's env variables

Reply:
I was trying out an approach that used a new env-vars.json file to store the client secret but maybe just setting the env var in the terminal is better? This way, there is no risk in accidentally committing a file with the client secret and we don't have to read in a file.